### PR TITLE
[MRG] DOC: update badges in README.rst

### PR DESCRIPTION
**Pull request by rythorpe**
I just noticed that our test badge in README.rst was still referencing Travis. I've now updated it to reference our CI with GH Actions. Here's what our compiled README.rst looks like now:

image

### CODE REVIEW

**Comment by rythorpe** [comment]
We could also add a badge for PyPI downloads if we want (see [MNE-Python](https://github.com/mne-tools/mne-python) as an example)....

**Comment by jasmainak** [comment]
Yes! And a binder-badge like here for the GUI: https://github.com/binder-examples/voila ... it can always be migrated to a new repo afterwards.

**Comment by jasmainak** [comment]
needs a rebase ... I guess because you built off the previous pull request

File: README.rst [hunk 1]
```diff
@@ -1,21 +1,26 @@
...
 	:target: https://codecov.io/gh/jonescompneurolab/hnn-core
-	:alt: Test coverage
 
-.. image:: https://user-images.githubusercontent.com/11160442/178095018-35d2619a-6a82-4e27-91c9-ff2796fab435.png
-   :target: https://user-images.githubusercontent.com/11160442/178095018-35d2619a-6a82-4e27-91c9-ff2796fab435.png
-   :alt: HNN-core GUI
+.. |PyPI| image:: https://img.shields.io/pypi/dm/hnn-core.svg?label=PyPI%20downloads
+	:target: https://pypi.org/project/hnn-core/
+
+.. |voila-binder| image:: https://mybinder.org/badge_logo.svg
```
Comment (jasmainak) [inline README.rst:20]: this won't work. You should have something like `/voila/render/hnn_widget.ipynb`. See:

https://voila.readthedocs.io/en/stable/deploy.html#deployment-on-binder

Comment (rythorpe) [reply README.rst:20]: Hmm, I've tried a bunch of things and can't get it to find the right path.

Comment (rythorpe) [reply README.rst:20]: Their docs aren't very clear.

**Comment by rythorpe** [comment]
> If you open the notebook without voila ... does the ipython notebook run?

Nope, it can't find the "hnn_core module"

**Comment by jasmainak** [comment]
missing from this list: https://github.com/jonescompneurolab/hnn-core/blob/master/binder/requirements.txt :-)

**Comment by rythorpe** [comment]
> missing from this list: https://github.com/jonescompneurolab/hnn-core/blob/master/binder/requirements.txt :-)

Isn't it suppose to run off of master branch? I had assumed that's what [postBuild](https://github.com/jonescompneurolab/hnn-core/blob/master/binder/postBuild) was for.

**Comment by jasmainak** [comment]
humm ... indeed. Something is going wrong. Maybe because you're inside inside `hnn_core/gui` folder? Some smart debugging is needed here

**Comment by jasmainak** [comment]
is this ready for merge @rythorpe ? We can remove the Binder badge if that's too much of a hassle to debug

**Comment by rythorpe** [comment]
If we're cool with waiting on the Binder badge for now, I can get this ready to merge ASAP.

**Comment by rythorpe** [comment]
This is a minor nitpick, but I just noticed that our sphinx cross-reference doesn't get built when GH renders the README.rst on our [repo webpage](https://github.com/jonescompneurolab/hnn-core):

image

How do we feel about this?

**Comment by jasmainak** [comment]
argh ... that may be an argument for using direct links as opposed to cross-links in certain locations.

**Comment by jasmainak** [comment]
is this ready for merge?

**Comment by rythorpe** [comment]
Any preference on whether I fix the contributing guide link here or in a different PR?

**Comment by jasmainak** [comment]
Merged after squash. Thanks @rythorpe !

**Pull request merged**